boards/arm/nucleo_f303k8/nucleo_f303k8.dts

```diff
@@ -44,14 +44,15 @@
 	status = "okay";
 };
 
-&clk_hsi {
+&clk_hse {
 	status = "okay";
+	clock-frequency = <DT_FREQ_M(8)>;
```

Comment:
please, Confirm if the HSE is Bypassed or not.

Reply:
Moved to HSI as no HSE on this board